6_custom/30_conclusion.tex

```diff
@@ -2,17 +2,10 @@
 
 Розроблено алгоритм розв’язування $(\max ,+)$ задач розмітки із
 цілочисельними якостями на основі субградієнтного спуску.
-Наведений алгоритм потребує не більше, ніж $|T|\cdot \log_2 |K| + 1$
-запусків субградієнтного спуску,
-на відміну від відомого алгоритму \cite{diffusion_shlezinger},
-для якого може знадобитись $|T|\cdot|K|$ викликів алгоритму дифузії.
 Зауважимо, що замість субградієнтного спуску можна використовувати
```

Comment:
Sorry, I cannot go to the final conclusion. This result is very valuable to be removed, so leave it
```suggestion
Наведений алгоритм потребує не більше, ніж $|T|\cdot \log_2 |K| + 1$
запусків субградієнтного спуску,
на відміну від відомого алгоритму \cite{diffusion_shlezinger},
для якого може знадобитись $|T|\cdot|K|$ викликів алгоритму дифузії.
Зауважимо, що замість субградієнтного спуску можна використовувати
```